src/base/net/downloadmanager.cpp

```diff
@@ -210,6 +212,15 @@ bool Net::DownloadManager::deleteCookie(const QNetworkCookie &cookie)
     return static_cast<NetworkCookieJar *>(m_networkManager.cookieJar())->deleteCookie(cookie);
 }
 
+bool Net::DownloadManager::hasSupportedScheme(const QString &url)
+{
+    const QStringList schemes = instance()->m_networkManager.supportedSchemes();
+    return std::any_of(schemes.cbegin(), schemes.cend(), [&url](const QString &scheme)
+    {
+        return url.startsWith((scheme + QLatin1Char(':')), Qt::CaseInsensitive);
```

Comment:
(paranoid) Shouldn't we filter out `magnet` scheme? In the highly unlikely case that Qt in the future can handle magnet URIs...

Reply:
Errr, I can't say much about this... since it is very unlikely to happen I would leave it out for simple code.

Reply:
I thought so. Consider this resolved.